packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -1190,27 +1190,27 @@ You can publish a theme for a page or email template with different combinations
 
 | Enum Value                      | Description                                                                                          |
 | ------------------------------- | ---------------------------------------------------------------------------------------------------- |
-| `OKTA_DEFAULT`                  | Use the Okta default logo with no background image and the Okta default colors on the Okta Sign-In Page.   |
-| `BACKGROUND_SECONDARY_COLOR`    | Use the logo from Theme with the `secondaryColorHex` as the background color for the Okta Sign-In Page.  |
-| `BACKGROUND_IMAGE`              | Use the logo and background image from Theme.                                                            |
+| `OKTA_DEFAULT`                  | Use the Okta default logo and Okta favicon with no background image and the Okta default colors on the Okta Sign-In Page.   |
```

Comment:
since we are using `default` with logo, how about using that with favicon as well? or remove from logo for consistency?

Reply:
thanks, i removed `default` everywhere to make it simpler and consistent